sdk/pom.xml

```diff
@@ -686,6 +686,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
Given that this is limited to test scope, we should be OK adding this dependency. Further, we already depend on guava, so pulling in the associated testing utilities makes sense.